e2e/cmd/run_tests/05_test_pam_krb5cc/main.go

```diff
@@ -0,0 +1,109 @@
+// Package main provides a script that runs PAM krb5cc-related tests on the
+// provisioned Ubuntu client.
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/ubuntu/adsys/e2e/internal/command"
+	"github.com/ubuntu/adsys/e2e/internal/inventory"
+	"github.com/ubuntu/adsys/e2e/internal/remote"
+)
+
+var sshKey string
+
+func main() {
+	os.Exit(run())
+}
+
+func run() int {
+	cmd := command.New(action,
+		command.WithValidateFunc(validate),
+		command.WithRequiredState(inventory.ADProvisioned),
+	)
+	cmd.Usage = fmt.Sprintf(`go run ./%s [options]
+
+Perform PAM krb5cc-related tests on the provisioned Ubuntu client.
+
+The runner must be connected to the ADSys E2E tests VPN.`, filepath.Base(os.Args[0]))
+
+	return cmd.Execute(context.Background())
+}
+
+func validate(_ context.Context, cmd *command.Command) (err error) {
+	sshKey, err = command.ValidateAndExpandPath(cmd.Inventory.SSHKeyPath, command.DefaultSSHKeyPath)
+	fmt.Println(sshKey)
```

Comment:
shouldn’t we print only after the error is not nil? I think you want to check that we don’t print anything if we return an error, but I think this is a more targetted test for it, wdyt?